data/network/src/androidMain/kotlin/social/androiddev/common/network/UrlStream.kt

```diff
@@ -6,6 +6,6 @@ import io.ktor.client.engine.cio.CIO
 import io.ktor.client.request.get
 import java.io.ByteArrayInputStream
 
-actual suspend fun urlStream(url: String): ByteArrayInputStream = HttpClient(CIO).use {
-    ByteArrayInputStream(it.get(url).body())
+suspend fun String.toUrlStream(): ByteArrayInputStream = HttpClient(CIO).use {
```

Comment:
why this change? Now it is not multiplatform anymore, right?

Reply:
It is now platform specific for Android/Jvm. Somewhere in `:ui` it is a shared source (since there are only Android/Jvm).

Reply:
but with the `expected/actual` pattern of KMP you can have different implementations for all platforms.
with your change it is just a function without taking care of KMP

Reply:
This breaks ios in that layer, therefor it cannot be a shared source for all. But it can in `ui`.